src/style.css

```
@@ -0,0 +1,127 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --color-search-bar--border: #e1e7ed;
+  --color-input-and-placeholder: #3d4e61;
+  --color-background-color: #fff;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  margin: 20px 0 0;
+  position: relative;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.input-big {
+  height: 70px;
+  width: 100%;
+  text-indent: 62px;
+  font-size: 16px;
+  font-weight: 300;
+  color: var(--color-input-and-placeholder);
+  font-family: Avenir-Book;
+  outline: none;
+}
+
+.input-big--appearance {
+  border-radius: 4px;
+  border: 1px solid hsl(210, 25%, 91%);
+  background: var(--color-background-color);
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.input-big:hover {
+  border-radius: 4px;
+  border: 1px solid var(--color-search-bar--border);
+  background: var(--color-background-color);
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.input-big:focus {
+  border-radius: 3px;
+  border: 1px solid var(--color-search-bar--border);
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.input-big::placeholder {
+  font-family: Avenir-Book;
+  font-weight: 400;
+  font-size: 16px;
+  color: var(--color-input-and-placeholder);
+}
+
+.icon-big {
+  height: 19px;
+  width: 19px;
+  position: absolute;
+  content: "";
+  top: 25px;
+  left: 26px;
+}
+
+.input-small {
+  height: 40px;
+  width: 100%;
+  text-indent: 33px;
+  font-size: 14px;
+  font-weight: 300;
+  color: var(--color-input-and-placeholder);
+  font-family: Avenir-Book;
+  padding: 0;
+  outline: none;
+}
+
+.input-small--appearance {
+  border-radius: 4px;
+  border: 1px solid var(--color-search-bar--border);
+  background: var(--color-background-color);
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.input-small:hover {
+  border-radius: 4px;
+  border: 1px solid var(--color-search-bar--border);
+  background: var(--color-background-color);
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.input-small:focus {
+  border-radius: 3px;
+  border: 1px solid var(--color-search-bar--border);
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.input-small::placeholder {
+  color: var(--color-input-and-placeholder);
+  font-family: Avenir-Book;
+  font-size: 14px;
+  font-weight: 300;
+}
+
+.icon-small {
```

```suggestion
.icon--small {
  height: 11px;
  width: 11px;
  top: 15px;
  left: 13px;
}
```

rest is redundant bcs we have this styles in `.icon` class